README.md

```diff
@@ -49,6 +49,13 @@ CANVAS_BASE_URL=http://your.server/
 CANVAS_ACCESS_TOKEN=your_token
 ```
 
+### Client specific
+We need to provide client specific env variables:
+```
+CLIENT_NAME=ClientName
+CANVAS_LOADER_ADMINISTRATOR_URL=https://clientname.com
```

Comment:
Is this the URL that they need to visit on our website to add a key? If that's the case, can't we use a relative path?

Reply:
GUIDE_FOR_CANVAS_INSTANCE_ADMNISTRATOR_URL

NB: the "administrator" here is the district/school employee who is the administrator of the canvas instance

Reply:
changed